contracts/package.json

```diff
@@ -48,6 +48,7 @@
     "@types/mocha": "^10.0.6",
     "@types/node": "^20.11.0",
     "@types/snarkjs": "^0.7.7",
+    "big-integer": "^1.6.52",
```

Comment:
Why do you need this dep?

Reply:
once you remove `zkey-manager` building starts failing, and it looks like it's due to this dependency: https://github.com/weijiekoh/optimisedmt -> says that circomlibjs@0.0.8 needs big-integer

Reply:
somehow that was shared by `zkey-manager`, and having it as dev dependency is enough.

Reply:
@ctrlc03 I think we can add this dep where optimisedmt is using (crypto package). If it doesn't work let's leave it.

Reply:
done!

Reply:
Getting this error in clrfund when running the contract unit tests, can you add `big-integer` as `dependencies` instead of `devDependencies`?

Error: Cannot find module 'big-integer'
Require stack:
- /Users/yuetloo/projects/clrfund/monorepo/contracts/node_modules/optimisedmt/node_modules/circomlibjs/src/smt_hashes_mimc.js
- /Users/yuetloo/projects/clrfund/monorepo/contracts/node_modules/optimisedmt/node_modules/circomlibjs/index.js
- /Users/yuetloo/projects/clrfund/monorepo/contracts/node_modules/optimisedmt/build/OptimisedMT.js
- /Users/yuetloo/projects/clrfund/monorepo/contracts/node_modules/optimisedmt/build/index.js
- /Users/yuetloo/projects/clrfund/monorepo/contracts/node_modules/maci-crypto/build/ts/AccQueue.js
- /Users/yuetloo/projects/clrfund/monorepo/contracts/node_modules/maci-crypto/build/ts/index.js
- /Users/yuetloo/projects/clrfund/monorepo/contracts/tests/deployer.ts

Reply:
sure thing will add